include/seqan3/alphabet/composition.hpp

```diff
@@ -182,8 +182,8 @@ struct alphabet_composition :
         using seqan3::to_rank;
         if constexpr (sizeof...(idx) > 0)
         {
-            return to_rank(get<0>(*this)) +
-                   ((to_rank(get<idx + 1>(*this)) * cummulative_alph_sizes[idx]) + ...);
+            return static_cast<rank_type>(to_rank(get<0>(*this)) +
```

Comment:
this gets more difficult to read now. Put it in one line if it fits, or  indent the `to_rank` calls so they align.